tests/functional/GNUmakefile

```diff
@@ -10,7 +10,7 @@ distclean maintainer-clean:
 	# includes, which are swiped when processing one subdir and then
 	# missing, as a hard error, for the other)
 	@$(MAKE) -C log_external $@
-	@mkdir .deps
+	@mkdir .deps 2>/dev/null || true
```

Comment:
You likely want `mkdir -p` instead of suppressing permission issues etc.

Reply:
good point